### PR TITLE
Upgrade simple-option to 0.14

### PR DESCRIPTION
**Pull request by orkunkl**
Related to https://github.com/CosmWasm/cosmwasm-examples/issues/121

### CODE REVIEW

**Review by maurolacy — APPROVED** [review]
lgtm. There are a couple of reference to `handle` and `init` in the docs (`Developing.md` and `Importing.md`) that you would want to change too.

**Comment by webmaster128** [comment]
> There are a couple of reference to `handle` and `init` in the docs (`Developing.md` and `Importing.md`) that you would want to change too.

I think it's better to delete them. Maintaining them in each contract folder is an impossible task.

**Comment by orkunkl** [comment]
@webmaster128 those markdown files, come from templates. I will fix it there but you are right these are unnecessary, will delete.

File: contracts/simple-option/src/contract.rs [hunk 1]
```diff
@@ -22,56 +22,57 @@ pub fn init(
...
-    state.owner = recipient.clone();
+    state.owner = deps.api.addr_validate(&recipient)?;
     config(deps.storage).save(&state)?;
 
-    let mut res = Context::new();
+    let mut res = Response::new();
     res.add_attribute("action", "transfer");
     res.add_attribute("owner", recipient);
-    Ok(res.into())
+    Ok(res)
```
Comment (webmaster128) [inline contracts/simple-option/src/contract.rs:68]: So much simpler now :)

**Pull request merged**